docs/en/ingest-management/fleet/air-gapped.asciidoc

```diff
@@ -40,9 +40,10 @@ a selection of packages.
 There are different distributions available:
 
 * {version} (recommended): +docker.elastic.co/package-registry/distribution:{version}+ - Selection of packages from the production repository released with the {version} version of the {stack}.
+* lite {version}: +docker.elastic.co/package-registry/distribution:lite-{version}+ - Subset of the most commonly used packages from the production repository released with the {version} version of the {stack}.
 * production: `docker.elastic.co/package-registry/distribution:production` - Packages available in the production registry (https://epr.elastic.co).
-* staging: `docker.elastic.co/package-registry/distribution:staging` - Packages available in the staging registry (https://epr-staging.elastic.co). These packages may be pending of validation.
-* snapshot: `docker.elastic.co/package-registry/distribution:snapshot` - Packages under development.
+* lite: `docker.elastic.co/package-registry/distribution:lite` - Small subset of packages available in the production registry (https://epr.elastic.co).
```

Comment:
Shouldn't this description be consistent with the one that we provide for the versioned distro? I'm not sure we provide enough info here for users to decide between the versioned and non-versioned distro.

Reply:
I'll update both definitions to be consistent

Reply:
About the difference between versioned and non-versioned distributions, it should be like in production. The packages that can be found in the versioned one are the ones that were  available at the moment of the Elastic stack X.Y release (e.g. 8.5.0). And the non-versioned contain the latest packages published.

Could we add something else to differentiate more those two distributions ?